### PR TITLE
qa/tests added mimic-x-singleton suite to rados as symlink

### PR DESCRIPTION
**Pull request by yuriw**
replaces https://github.com/ceph/ceph/pull/27131

Signed-off-by: Yuri Weinstein <yweinste@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

**Comment by yuriw** [comment]
testing as:

CEPH_QA_MAIL="ceph-qa@lists.ceph.com"; MACHINE_NAME=smithi; CEPH_BRANCH=nautilus

teuthology-suite -v --ceph-repo https://github.com/ceph/ceph.git --suite-repo https://github.com/yuriw/ceph.git -c $CEPH_BRANCH -m $MACHINE_NAME -s rados/upgrade -k distro -p 90 -e $CEPH_QA_MAIL --suite-branch wip-yuriw-38845-nautilus

http://pulpito.front.sepia.ceph.com:80/yuriw-2019-03-28_22:31:06-rados:upgrade-nautilus-distro-basic-smithi/

**Comment by yuriw** [comment]
I added `qa/releases/octopus.yaml`, assuming we will have it:
```
tasks:
- exec:
    osd.0:
      - ceph osd require-osd-release octopus
      - ceph osd set-require-min-compat-client octopus
- ceph.healthy:

```

**Comment by yuriw** [comment]
http://pulpito.front.sepia.ceph.com/yuriw-2019-03-28_22:31:06-rados:upgrade-nautilus-distro-basic-smithi/

File: qa/releases/octopus.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+tasks:
```
Comment (neha-ojha) [inline qa/releases/octopus.yaml]: we should be using nautilus.yaml

**Comment by yuriw** [comment]
http://pulpito.ceph.com/yuriw-2019-03-29_19:33:37-rados:upgrade-nautilus-distro-basic-smithi/

**Pull request merged**